### PR TITLE
Android release note catchup

### PR DESCRIPTION
**Pull request by bdwyer-newrelic**
Catching up on release notes for v5.4.0, v5.5.0 and v5.7.0 for New Relic's android app

### CODE REVIEW

File: src/content/docs/release-notes/mobile-apps-release-notes/new-relic-android-release-notes/new-relic-android-5700.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+subject: Mobile app for Android
+releaseDate: '2023-08-02'
+version: 5.7.0
+downloadLink: 'https://play.google.com/store/apps/details?id=com.newrelic.rpm'
+---
+
+### Notes
+
+Now with support for Log Partitions
```
Comment (ally-sassman) [inline src/content/docs/release-notes/mobile-apps-release-notes/new-relic-android-release-notes/new-relic-android-5700.mdx:10]:
```suggestion
Now with support for [Log Partitions](https://docs.newrelic.com/whats-new/2021/04/log-part/).
```

File: src/content/docs/release-notes/mobile-apps-release-notes/new-relic-android-release-notes/new-relic-android-5500.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+releaseDate: '2023-07-20'
+version: 5.4.0
+downloadLink: 'https://play.google.com/store/apps/details?id=com.newrelic.rpm'
+---
+
+### Notes
+
+- New Tags screen for entities
+- More detailed charts and information for Kubernetes entities
+- Improved favorites and sorting for Infrastructure
```
Comment (ally-sassman) [inline src/content/docs/release-notes/mobile-apps-release-notes/new-relic-android-release-notes/new-relic-android-5500.mdx:12]:
```suggestion
- Improved favorites and sorting for infrastructure monitoring
```

File: src/content/docs/release-notes/mobile-apps-release-notes/new-relic-android-release-notes/new-relic-android-5400.mdx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+subject: Mobile app for Android
+releaseDate: '2023-06-25'
+version: 5.5.0
+downloadLink: 'https://play.google.com/store/apps/details?id=com.newrelic.rpm'
+---
+
+### Notes
+
+Now with an updated Infrastructure experience!
```
Comment (ally-sassman) [inline src/content/docs/release-notes/mobile-apps-release-notes/new-relic-android-release-notes/new-relic-android-5400.mdx:10]:
```suggestion
Updated the infrastructure monitoring experience.
```

**Review by ghost — CHANGES_REQUESTED** [review]
See my comments

File: src/content/docs/release-notes/mobile-apps-release-notes/new-relic-android-release-notes/new-relic-android-5400.mdx (hunk 3 above)
Comment (ally-sassman) [reply src/content/docs/release-notes/mobile-apps-release-notes/new-relic-android-release-notes/new-relic-android-5400.mdx:10]: @bdwyer-newrelic Is this accurate, or is it referring to a specific infrastructure feature?

Comment (bdwyer-newrelic) [reply src/content/docs/release-notes/mobile-apps-release-notes/new-relic-android-release-notes/new-relic-android-5400.mdx]: Yep! That is accurate. The mobile app is always updating to reflect the web. This release was another instance of this.

**Comment by bdwyer-newrelic** [comment]
Looks like you applied the changes, so I re-requested a review from...yourself? Is that the next step in this workflow?

**Review by ghost — APPROVED** [review]

**Pull request merged**